### DISCUSSION TITLE
Is Svelte supported?

### DISCUSSION THREAD

**Discussion by vdawg-git** [oscartbeaumont/rspc#255] [rspc Help] [answered]
Judging from the docs, one needs to use the Vanilla core of the project.
Is this recommended?

**Comment by oscartbeaumont** [comment, marked as answer]
Partially. We have started work on the client but it's not as ready as the other clients.

Will hopefully have better support in the near future as we are reworking the Tanstack integration so we don't need to maintain a package for each framework.
